contracts/ERC721M.sol

```diff
@@ -17,15 +17,15 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
 
     bool private _mintable;
     string private _currentBaseURI;
-    uint256 private _activeStage;
+    uint256 internal _activeStage;
     uint256 private _maxMintableSupply;
     uint256 private _globalWalletLimit;
     string private _tokenURISuffix;
     bool private _baseURIPermanent;
     address private _cosigner;
     address private _crossmintAddress;
 
-    MintStageInfo[] private _mintStages;
+    MintStageInfo[] internal _mintStages;
```

Comment:
same here, prefer getter function so that we don't leak private access to other contracts